.pipelines/upgrade-e2e-job.yaml

```diff
@@ -10,9 +10,9 @@ pr:
   branches:
     include:
     - main
-  paths:
-    include:
-      - charts/osm-arc/Chart.yaml
+  #paths:
+    #include:
+      #- charts/osm-arc/Chart.yaml
```

Comment:
why dis commented out

Reply:
Will recomment before merging, just wanted to run the upgrade job in this PR to ensure it works with pre-install validation.